docs/faq.md

```diff
@@ -756,9 +750,7 @@ There are some common reasons why containers fail on launch.
 
 #### My shell script won't run
  
-The default base image used by Jib, ([`distoless/java`](https://github.com/GoogleContainerTools/distroless/tree/master/java)), does not include a shell, and thus shell scripts won't launch.
-
-Solution: use a different base image with a shell.
+The default base image used by the Jib Maven and Gradle plugins prior to 3.0 did not have a shell, and thus shell scripts won't launch. See [Where is bash?](#where-is-bash) for more details.
```

Comment:
maybe something like this?
```suggestion
Jib Maven and Gradle plugins prior to 3.0 used Distroless Java as the default base image, which does not have a shell. See [Where is bash?](#where-is-bash) for more details.
```

Reply:
Actually, do we need this FAQ if we have the "Where is bash?" one

Reply:
We can take it out maybe a year later. It'll take a lot of time before the majority of people upgrade to 3.0, so I think it's possible that they'll hit this issue.